packages/libs/components/src/plots/ExportPlotToImageButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { Image } from '@material-ui/icons';
+import { FloatingButton } from '@veupathdb/coreui';
+
+interface ToImageOpts {
+  height: number;
+  width: number;
+  format: 'svg' | 'png';
+}
+
+interface ToImage {
+  (opts: ToImageOpts): Promise<string>;
+}
+
+interface Props {
+  filename?: string;
+  toImage: ToImage;
+}
+
+export function ExportPlotToImageButton(props: Props) {
+  const { filename = 'plot', toImage } = props;
+  const ext = 'svg';
+  return (
+    <div
+      style={{
+        width: '100%',
+        display: 'flex',
+        justifyContent: 'end',
+        padding: '.5em 0',
+      }}
+    >
+      <FloatingButton
+        onPress={() => downloadPng(toImage, filename, ext)}
+        text={`Export to ${ext.toUpperCase()}`}
+        icon={Image}
+        themeRole="primary"
+        textTransform="none"
+        size="small"
+      />
+    </div>
+  );
+}
+
+async function downloadPng(
+  toImage: ToImage,
+  filename: string,
+  ext: ToImageOpts['format']
+) {
+  const imgUrl = await toImage({ height: 450, width: 750, format: ext });
```

Comment:
just a note - initiall i was wary of setting the height and width because the bipartite networks have a height based on the number of nodes. However i gave it a shot with a super tall network and it worked great!

Reply:
Okay, great! We can make this more configurable later, if needed.